packages/ecosystem/src/zod/zod.spec.ts

```diff
@@ -45,10 +46,10 @@ test('zod schemas are supported', async () => {
   expect(handler).not.toHaveBeenCalled();
 });
 
-test('collects multiple errors per field', async () => {
+skip('collects multiple errors per field', async () => {
```

Comment:
use `test.skip`, but moreover why do we need to skip tests here?

Reply:
Ah sorry, checked your PR description again and it makes sense. Just use `test.skip`